rust/chains/hyperlane-sealevel/src/mailbox.rs

```diff
@@ -426,15 +546,29 @@ impl Mailbox for SealevelMailbox {
             .as_ref()
             .ok_or_else(|| ChainCommunicationError::SignerUnavailable)?;
 
-        let mut instructions = Vec::with_capacity(2);
+        let mut instructions = Vec::with_capacity(4);
```

Comment:
is this change still required? Looks like regardless of the if-else branches below, the same number of instructions gets pushed into `instructions`

Reply:
changed to 3 - the 2 was incorrect originally